userspace/libsinsp/test/public_sinsp_API/interesting_syscalls.cpp

```diff
@@ -0,0 +1,435 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+*/
+
+#include <gtest/gtest.h>
+#include <sinsp.h>
+#include <sys/syscall.h>
+#include "../test_utils.h"
+
+std::set<uint32_t> ordered_sinsp_state_ppm_sc_set{
```

Comment:
is this a test set or does this need to change if new syscalls are added?

Reply:
unfortunately, this set must change if we change the sinsp state internal set, this is precisely the scope of this test, checking that we are considering the right events. Maybe i can add a comment to this

Reply:
BTW in the next future, I think we will add some further automation around the interesting syscall topic, this is only a first check to ensure that all works smooth

Reply:
Definitely better to add a comment. Meaning, if someone adds a new event this test will break and it needs to be clear to understand what you should do.